app/scripts/modules/core/delivery/service/executions.transformer.service.js

```diff
@@ -299,7 +299,7 @@ module.exports = angular.module('spinnaker.core.delivery.executionTransformer.se
         let jenkins = deploymentDetails[0].jenkins;
         execution.buildInfo = {
           number: jenkins.number,
-          url: `${jenkins.host}job/${jenkins.name}/${jenkins.number}`
+          url: deploymentDetails[0].buildInfoUrl || `${jenkins.host}job/${jenkins.name}/${jenkins.number}`
```

Comment:
The tags on the images are set without jenkins in the scope, but clouddriver puts them into the jenkins object https://github.com/spinnaker/clouddriver/blob/5eaff47add8ae028daf35b3f19dce2da3e3a58f8/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonClusterProvider.groovy#L224

So this is right.